src/components/general/AddStakeholderModal/styles.scss

```diff
@@ -0,0 +1,31 @@
+.modal {
+    display: flex;
+    width: 80vw;
+    height: 90vh;
+
+    .heading {
+        color: var(--dui-color-brand);
```

Comment:
This should not be styled from here
CC: @AdityaKhatri

Reply:
@AdityaKhatri any idea on what needs to be done here?

Reply:
Yes. @samshara . Lets not style it and let it be its normal form.
We can style it from Modal later on.